posts/2018-01-07-how-to-read.md

```diff
@@ -0,0 +1,46 @@
+---
+title: How to Read
+---
+
+I've been told the rate and retention I have while reading is abnormal to say
+the least.  Others have been interested in trying my reading methodology and
+seeing if it would help them read better.  As a result, I've been making an
+attempt to codify how I read so that others can try it for themselves.
+
+The first idiosyncracy to my methodology is the concurrency I utilize while
+reading books.  Since college I've always read around five to ten books
+concurrently.  Generally, this is received with awe and intrigue about how I'm
+able to keep them separate or retain any number of them let alone one.
+
+What I've found over the years is that by reducing the amount of information
```

Comment:
Change this sentence to: Over the years I've found that reducing the amount of information consumed on a topic in a given sitting (subsequently?)  increases the ability to let that knowledge settle into long-term memory (references).